lib/src/main/java/fi/epicbot/toster/report/formatter/JsonFormatter.kt

```diff
@@ -0,0 +1,13 @@
+package fi.epicbot.toster.report.formatter
+
+import fi.epicbot.toster.report.model.ReportOutput
+import kotlinx.serialization.encodeToString
+import kotlinx.serialization.json.Json
+
+class JsonFormatter(private val prettyPrintJson: Boolean) : ReportFormatter {
+
+    override fun format(reportOutput: ReportOutput): String {
```

Comment:
Is it neccessary to have this parameter so dynamic? Maube pass it to constructor?

Reply:
I can't see any difference. with one formatter you can format different outputs